R/learner_survivalmodels_surv_dnnsurv.R

```diff
@@ -24,7 +24,13 @@
 #' `r format_bib("zhao2019dnnsurv")`
 #'
 #' @template seealso_learner
-#' @template example
+#' @examples
+#' packages <- c("mlr3extralearners", "survivalmodels", "keras", "pseudo", "tensorflow", "distr6")
```

Comment:
use = instead of <-

Also just for constructing the learner the packages dont have to be installed so all the keras, pseudo etc. can be removed from the requires_namespaces